release-notes/5.1.0.md

```diff
@@ -0,0 +1,319 @@
+  in the database.  The number of times is now a constant that can be set in
+  `civicrm.settings.php`.
+
+- **[CRM-21840](https://issues.civicrm.org/jira/browse/CRM-21840) Show Tool Icon
+  For Radio Button Groups
+  ([11819](https://github.com/civicrm/civicrm-core/pull/11819))**
+
+  A group of radio buttons will now display a wrench icon if the user has the
+  permission to edit the options.  This now matches the feature for select
+  boxes.
+
+- **[CRM-20981](https://issues.civicrm.org/jira/browse/CRM-20981) Allow custom
+  base-pages with less `crmApp` boilerplate
+  ([10783](https://github.com/civicrm/civicrm-core/pull/10783))**
+
+  Extensions that use AngularJS can now define a base path more easily.
+
+- **[CRM-20624](https://issues.civicrm.org/jira/browse/CRM-20624) Expose
+  description field for option values in UI
+  ([10403](https://github.com/civicrm/civicrm-core/pull/10403))**
+
+  Custom field options have a `description` field (that has been in the database
+  for a while); now it can be set from the user interface and viewed when
+  managing option values.
+
+- **[CRM-21835](https://issues.civicrm.org/jira/browse/CRM-21835) Detailed
+  Logging - Use INNODB if Archive is not available
+  ([11786](https://github.com/civicrm/civicrm-core/pull/11786))**
+
+  The Archive storage engine is not always available in MariaDB, even though it
+  is the default for log tables when detailed logging is enabled.  Now, CiviCRM
+  will fall back to InnoDB if Archive is not available.
+
+- **[CRM-21753](https://issues.civicrm.org/jira/browse/CRM-21753) Support
+  'criteria' in url on batch merge form
+  ([11666](https://github.com/civicrm/civicrm-core/pull/11666)) (completes prior
+  work)**
+
+  This improves handling of criteria set in the URL for batch merges.
+
+- **[CRM-20811](https://issues.civicrm.org/jira/browse/CRM-20811) Schedule
+  Reminders uses mixed terminology for Active state: Active? versus Send Email
+  ([10602](https://github.com/civicrm/civicrm-core/pull/10602))**
+
+  The option for whether a scheduled reminder is active is now displayed as a
+  checkbox with "Scheduled Reminder Active" rather than "Send Email" or "Send
+  email or SMS".
+
+- **VersionCheck - Get more nuanced messages from latest.civicrm.org
+  ([11991](https://github.com/civicrm/civicrm-core/pull/11991))**
+
+  Upgrade messages are now supplied by the CiviCRM version server rather than
+  being calculated by each site.
+
+### CiviContribute
+
+- **[CRM-20610](https://issues.civicrm.org/jira/browse/CRM-20610) Replace
+  payment details block with editable payment list on 'Edit Contribution' form
+  ([10776](https://github.com/civicrm/civicrm-core/pull/10776))**
+
+  The backend contribution edit form is now based on the same payment editing
+  interface as events with multiple payments.
+
+## <a name="bugs"></a>Bugs resolved
+
+### Core CiviCRM
+
+- **[CRM-21784](https://issues.civicrm.org/jira/browse/CRM-21784) View custom
```

Comment:
Resolving incomplete features is the real grey area on doing the release notes.  I had it down here because it seems like a bug if you can set a value on a field but never see it again.  However, I totally see your point that it's adding functionality.

Reply:
Yeah - & the other grey area is code cleanup - the cleaning up of joins on reports is probably arguably a bug fix. 

I guess in this case I think it fits into - would some people look at this list & think 'oh yeah I've been wanting that I'll upgrade'

Reply:
fair enough--moved it